lighthouse-core/gather/computed/metrics/consistently-interactive.js

```diff
@@ -28,8 +33,11 @@ class ConsistentlyInteractive extends MetricArtifact {
    */
   static _findNetworkQuietPeriods(networkRecords, traceOfTab) {
     const traceEndTsInMs = traceOfTab.timestamps.traceEnd / 1000;
+    /** @param {LH.WebInspector.NetworkRequest} record */
+    const filter = record => record.finished && record.requestMethod === 'GET' && !record.failed &&
+        Math.floor(record.statusCode / 100) < 4;
     return NetworkRecorder.findNetworkQuietPeriods(networkRecords,
-      ALLOWED_CONCURRENT_REQUESTS, traceEndTsInMs);
+      ALLOWED_CONCURRENT_REQUESTS, traceEndTsInMs, filter);
```

Comment:
making `NetworkRecorder.findNetworkQuietPeriods` polymorphic like this makes me nervous since we use it for monitoring network quiet during page load. In CLI usage it's no big deal since this always happens after gathering is finished, but wdyt about avoiding the question altogether and filter before passing in?

Reply:
filtering before we passing networkrecords seems reasonable.

Reply:
yeah sg, changed to filter in TTI